apps/web/src/pages/_document.tsx

```diff
@@ -12,6 +14,13 @@ export default function Document() {
           crossOrigin="anonymous"
         />
         <link rel="icon" href="/favicon.png" />
+        {plausibleDomain && (
+          <script
+            defer
+            data-domain={plausibleDomain}
+            src="https://plausible.io/js/script.js"
+          ></script>
+        )}
```

Comment:
I believe that the  `PlausibleProvider` in the `_app.tsx` file does this already